lib/utils/__tests__/validateOptions.test.js

```diff
@@ -15,66 +14,66 @@ describe("validateOptions for primary options", () => {
       possible: [ "a", "b", "c" ],
       actual: "a",
     })
-    expect(result.warn.calledOnce).toBeFalsy()
-    result.warn.reset()
+    expect(result.warn.mock.calls).toHaveLength(0)
+    result.warn.mockClear()
```

Comment:
I think that instead of using `mockClear` invocations maybe we should use [the `clearMocks` configuration option](https://facebook.github.io/jest/docs/en/configuration.html#clearmocks-boolean). Have you looked into that?